msalautomationapp/build.gradle

```diff
@@ -182,7 +182,7 @@ dependencies {
     compileOnly "org.projectlombok:lombok:$rootProject.ext.lombokVersion"
     annotationProcessor "org.projectlombok:lombok:$rootProject.ext.lombokVersion"
 
-    implementation(project(":testutils")) {
+    implementation(project(":LabApiUtilities")) {
         exclude module: 'common'
```

Comment:
I think we can remove this exclude now since LabApiUtilitities doesn't actually take a dependency on common